<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>660</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>华光淡古印_CNKI</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>24小时</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/image/notebook.png</normaloff>:/image/notebook.png</iconset>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>660</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光淡古印_CNKI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="table">
    <attribute name="title">
     <string>时间表</string>
    </attribute>
    <widget class="QScrollArea" name="tableScroll">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>45</y>
       <width>534</width>
       <height>585</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>534</width>
       <height>632</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tableAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>532</width>
        <height>630</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>532</width>
        <height>630</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="plusTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>5</y>
       <width>534</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华光淡古印_CNKI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>CrossCursor</cursorShape>
     </property>
     <property name="text">
      <string>增加一条</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="taboo">
    <attribute name="title">
     <string>禁忌</string>
    </attribute>
    <widget class="QLabel" name="tabooImage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>534</width>
       <height>200</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="plusTaboo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>205</y>
       <width>534</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华光淡古印_CNKI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>CrossCursor</cursorShape>
     </property>
     <property name="text">
      <string>增加一条</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="tabooScroll">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>245</y>
       <width>534</width>
       <height>387</height>
      </rect>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tabooAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>532</width>
        <height>385</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
